docs/source/guides/configuration.rst

```diff
@@ -101,6 +102,10 @@ Environment-specific parameters
   This parameter allows users to find an optimal value in a tradeoff between
   resource consumption and the delay of getting back to work after an idle
   period.
+- ``mst_stale_threshold_milliseconds`` is an optional parameter that sets the time after which the node that sent the MST transaction in the first place would try to send it again if no final status is received.
+  It is needed in case, say, some nodes failed and restarted or experienced network problems, so could not receive the transaction.
+  Default value is 1800000 (30 minutes).
+  We recommend not to set it to too small value – it will be using processing resources constantly connecting the nodes and also not to set it to too big value – that will cause delays in receiving MST transactions in case of failed nodes.
```

Comment:
> <...> of failed nodes

this also concerns failed network 

maybe rephrase "too (small/big)" with something more neutral, like "increasing this value leads to..."?